src/renderers/WebGLRenderer.js

```diff
@@ -1862,6 +1867,10 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 				refreshUniformsPhong( m_uniforms, material );
 
+			} else if ( material instanceof THREE.MeshPhysicalMaterial ) {
```

Comment:
Oh, I was saying that because I noticed `refreshUniformsPhysical()` was just calling `refreshUniformsStandard()` anyway.